channels/chan_usbradio.c

```diff
@@ -431,7 +433,8 @@ static struct chan_usbradio_pvt usbradio_default = {
 	.area = 0,
 	.rptnum = 0,
 	.clipledgpio = 0,
-	.rxaudiostats.index = 0
+	.rxaudiostats.index = 0,
+	.legacyaudioscaling = 1,
```

Comment:
Same here, comment that this setting will be disabled by default or removed eventually and make it a bitfield